dealiiadapter.cc

```diff
@@ -770,6 +791,124 @@ void ElasticProblem<dim>::output_results(const unsigned int timestep) const
     std::cout<< "\t Output written to solution-" + std::to_string(timestep) + ".vtk \n" <<std::endl;
 }
 
+template <int dim>
+void ElasticProblem<dim>::configure_precice()
+{
+    precice.configure(parameters.config_file);
+    meshID = precice.getMeshID(parameters.mesh);
+    read_data_ID  = precice.getDataID(parameters.read_data_name, meshID);
+    write_data_ID = precice.getDataID(parameters.write_data_name, meshID);
```

Comment:
is there a reason to make configure and initialize two separate functions? 
if so, normally we would do `mesh_id`, `data_id`, ... in initialize